pkg/expr/expr.go

```diff
@@ -150,9 +151,14 @@ func newConstant(v string, vType config.ValueType) (*Constant, error) {
 			return nil, err
 		}
 	default: // string
```

Comment:
I agree that this feels like a separate PR. However, we don't need those missing config types yet... and this PR is about restoring functionality that was working before, so I feel like it is OK to include this little bit in this PR.  Sound ok?